src/main/kotlin/no/nav/helse/sykmelding/domain/Sykmelding.kt

```diff
@@ -21,22 +23,39 @@ data class Sykmelding(val jsonNode: JsonNode) {
     val id = jsonNode["id"].asText()!!
 
     val aktørId = jsonNode["pasientAktoerId"].asText()!!
+
+    val syketilfelleStartDato: LocalDate? = jsonNode["syketilfelleStartDato"].safelyUnwrapDate()
+    val perioder: List<Periode> = jsonNode["perioder"].map {
+        Periode(
+            fom = LocalDate.parse(it["fom"].textValue()),
+            tom = LocalDate.parse(it["tom"].textValue())
+        )
+    }
```

Comment:
her burde `Periode` også vært et embedded document, og tatt inn `it` som argument.
burde også brukt lazy her, sånn at vi ikke mapper til `Periode` uten at den blir kalt.

à la: 
```
val perioder get() = jsonNode["perioder"].map { Periode(it) }
```